ui/raidboss/data/02-arr/raid/t6.js

```diff
@@ -265,12 +264,15 @@
         'Thorn Whip': 'Dornenpeitsche',
         'Viscid Emission': 'Klebsporen',
       },
+      '~effectNames': {
+        'Honey-Glazed': 'Honigsüß',
+      },
     },
     {
       'locale': 'fr',
       'replaceSync': {
+        'Rafflesia': 'rafflesia',
```

Comment:
Rafflesia 
Thank you

Reply:
If it helps you not have to spend so much time looking at these, `replaceSync` never appears to users and is always case insensitive.  So, `Rafflesia` and `rafflesia` are functionally identical.  `replaceText` appears on timelines, so it's important to capitalize it the way you'd like it to be displayed, but `replaceSync` is only used for regular expressions.